action/sealedenvelope.go

```diff
@@ -22,6 +23,20 @@ type SealedEnvelope struct {
 	evmNetworkID uint32
 	srcPubkey    crypto.PublicKey
 	signature    []byte
+	address      address.Address
+	hash         *Hash256Result
+}
+
+type Hash256Result struct {
```

Comment:
`hash256Result`, this is internal data struct

Reply:
Remove this struct